### ISSUE TITLE
Line 312-313 of BASE_MODEL.py - Should we use bid_len instead of market_len? 

### ISSUE THREAD

**Issue by Seondong** [rk2900/drsa#1]
Hello Kan, I have a difficulty of mapping between the loss function in the paper and the code. Up to this point, I understood the variable 'cost' in BASE_MODEL.py refers L_c in, and the variable 'anlp_node' refers L_z. 

As explained in the paper (page 4), L_z is a negative log likelihood of true event time z = z^i over the uncensored logs Pr(z \in V_{l^i}), which is only calculated the 'win' case. 

Here is my question. In the line 312 of BASE_MODEL.py, where you define two variable, is it right to use x[0:market_len+1], x[0:market_len] instead of x[0:bid_len+1], x[0:bid_len]?
 
(Line 312) anlp_rate_last_one = tf.reduce_prod(x[0:market_len + 1]) 
(Line 313) anlp_rate_last_two = tf.reduce_prod(x[0:market_len]) 
(Line 320) log_minus = tf.log(tf.add(tf.transpose(rate_result)[2] - tf.transpose(rate_result)[1], 1e-20))

I think we should use the difference between x[0:bid_len+1], x[0:bid_len] since 'bid_len' means the true_event time instead of 'market_len' which denotes an observation time, and the difference between two rate_rusult is a dying probability of that time period.  

Thank you,
Sundong (sundong.kim@kaist.ac.kr)
